integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.spec.ts

```diff
@@ -141,76 +140,54 @@ describe('ScheduleLinesCartEntryComponent', () => {
       });
 
       it("should contain div element with class name 'cx-visually-hidden' that contains a hidden schedule line info", function () {
-        CommonConfiguratorTestUtilsService.expectElementContainsA11y(
-          expect,
-          htmlElem,
-          'div',
-          'cx-visually-hidden',
-          0,
-          undefined,
-          undefined,
+        const divElementWithVisuallyHiddenClass = htmlElem.querySelector(
+          '.cx-visually-hidden'
+        );
+
+        expect(divElementWithVisuallyHiddenClass?.innerHTML).toContain(
           's4omScheduleLines.a11y.scheduleLineEntryInfo'
         );
       });
 
       it("should contain div element with 'cx-schedule-line-info' and aria-describedby attribute that refers to a corresponding attribute-value pair", () => {
-        CommonConfiguratorTestUtilsService.expectElementContainsA11y(
-          expect,
-          htmlElem,
-          'div',
-          'cx-schedule-line-info',
-          undefined,
-          'aria-describedby',
-          'cx-schedule-line-info-0'
+        const divElementWithScheduleLineInfoClass = htmlElem.querySelector(
+          '.cx-schedule-line-info'
         );
-      });
 
-      it("should contain div element with class name 'cx-visually-hidden' that refers to a corresponding attribute-value pair", () => {
```

Comment:
to the reviewer:

this deleted test assertion didn't even work `should contain div element with class name 'cx-visually-hidden' that refers to a corresponding attribute-value pair`

When I say, it doesn't work, I mean it's testing against nothing 'literally'.